documentation/first-steps/example-pom.xml

```diff
@@ -1,6 +1,6 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>org.heigit.bigspatialdata.oshdb-api-tutorial</groupId>
+  <groupId>org.heigit.ohsome.oshdb-api-tutorial</groupId>
```

Comment:
When discussing about this: is this example supposed to be in a different package and not a subpackage of `org.heigit.ohsome.oshdb`?